### ISSUE TITLE
react-map-gl-draw Editor Init features error

### ISSUE THREAD

**Issue by syk543277780** [uber/nebula.gl#305]
I am trying to init features for Editor

I got errors:
![Screen Shot 2019-12-15 at 11 41 16 PM](https://user-images.githubusercontent.com/28099395/70861561-87ab6900-1f94-11ea-9272-203214cfcb64.png)

here is my features value
```
this.geojsonData = [
      {
        "id": "5fd0b480-1f1c-11ea-bdd3-0b62bcfd0fac",
        "properties": { "renderType": "Polygon" },
        "geometry":
        {
          "type": "Polygon", 
          "coordinates":
            [
              [
              [174.11878660051664, -36.12043610070665],
              [174.12097528307285, -36.12426661606648],
              [174.13084581222716, -36.12430128050225],
              [174.13121059267368, -36.119066776120846],
              [174.11878660051664, -36.12043610070665]
            ]
          ]
        }
      }
    ]
```

here is my render
```
<ReactMapboxGl
        mapboxApiAccessToken={""}
        mapStyle="mapbox://styles/mapbox/satellite-v9"
        // mapStyle="mapbox://styles/mapbox/streets-v11"
        {...this.state.viewport}
        onViewportChange={this._updateViewport}
      >
        <Editor
          ref={_ => (this._editorRef = _)}
          style={{ width: '100%', height: '100%' }}
          clickRadius={12}
          mode={mode}
          features={this.geojsonData}
          onSelect={this._onSelect}
          onUpdate={this._onUpdate}
          editHandleShape={'circle'}
          featureStyle={getFeatureStyle}
          editHandleStyle={getEditHandleStyle}
        />
        {this._renderDrawTools()}
      </ReactMapboxGl>
```

is it something wrong with my features data?

**Comment by psmaAaron** [comment]
I've had the exact same problem.

Making a ref to the Editor component, then using that to add GeoJSON data works, but it's not a good workaround.

**Comment by xintongxia** [comment]
should be fix in v0.18.3. However v0.18.3 editor's API have changed significantly, please check the API [doc](https://github.com/uber/nebula.gl/tree/master/modules/react-map-gl-draw) and examples.


Feel free to reopen if you still see the issues.
